config/initializers/okcomputer.rb

```diff
@@ -61,9 +62,7 @@ def profile
 OkComputer::Registry.register 'suri_url', OkComputer::HttpCheck.new(Settings.SURI.URL)
 
 # Stacks
-OkComputer::Registry.register 'stacks_doc_cache_storage_root', OkComputer::DirectoryCheck.new(Settings.STACKS.DOCUMENT_CACHE_STORAGE_ROOT)
```

Comment:
IIRC (and i may be misremembering, it's been a while):  file content retrieval is now proxied through dor-services-app, and argo no longer does the SFTP thing.